dashboard/db/migrate/20160324173226_create_plc_course_units.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePlcCourseUnits < ActiveRecord::Migration
+  def change
+    create_table :plc_course_units do |t|
+      t.references :plc_course, index: true, foreign_key: true
+      t.string :unit_name
```

Comment:
Nit:  I think the unit_ is clear from context so you could just do "name", "description", and "order".  No need to change this unless you want to.